src/main/java/net/bootsfaces/component/inputText/InputTextRenderer.java

```diff
@@ -297,27 +294,19 @@ public void encodeEnd(FacesContext context, UIComponent component) throws IOExce
 						"}" + //
 						"});";//
 
-				// The following lines fix issue #1079 on tags with typeahead.
-				// They empty duplicated 'name' attribute on generated input (which holds the original HTML id).
-				String inputId = fieldId; // input id
-				inputId = inputId.replace(":", "\\\\:"); // escape the id for jQuery
-				js += "$('#" + inputId + "').attr('name','');";
+                                js +=  "$('." + escapedClientId + "').siblings('.bootstrap-tagsinput').addClass('form-control');";
 
```

Comment:
with the update to 0.8.0 the bug with the 'name' was fixed, but there is a new bug fix a missing class